hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,6 @@
+RELEASE_TYPE: patch
+
+This adds a new minimizer step, which reorders byte arrays: 3 1 2 becomes 1 2 3.
```

Comment:
We tend to avoid low-level details in release notes and just say "The shrinker got better" or words to that effect. e.g. https://hypothesis.readthedocs.io/en/latest/changes.html#v3-40-1